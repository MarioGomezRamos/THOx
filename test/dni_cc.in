# d+58Ni CDCC (no spins)
# ---------------------------------------------------------------------------------------------
 &SYSTEM  Zv=0. Av=1.0087   sn=0.5
          Zc=1. Ac=1.0078  egs=2.22/

 &corestates spin=0.5 parity=+1 ex=0.0 /
 &corestates /

 &output wfout(:)=1 5   xcdcc=F verb=3 solapout(:)=0  cdccwf=F  /

 &GRID  rmin=0.0 rmax=100.0 dr=0.05 rlast=100 /

# p-n potential
 &POTENTIAL ptype=8 ap=1 at=0
   vl0(0:2)=-72.15 -0 -0
   r0=0   a0=1.484  /

  &potential ptype=0 /

 &pauli n=0 /


  &JPSET bastype=6 nodes=1 mlst=4 gamma=1.5 bosc=1.6 nho=20 nsp=0 exmin=-5 exmax=25 bas2=0 JTOT=1.0 PARITY=1 lmax=2 changepot=.false./  l=0   /
!  &JPSET bastype=1 mlst=4 gamma=1.5 bosc=1.6 nho=20 nsp=0 exmin=0 exmax=25 bas2=0 JTOT=1.0 PARITY=1 lmax=2 / 
!  &JPSET bastype=1 mlst=4 gamma=1.5 bosc=1.6 nho=20 nsp=0 exmin=-5 exmax=10 bas2=0 JTOT=0.0 PARITY=1 lmax=0 /
!  &JPSET bastype=1 mlst=4 gamma=1.5 bosc=1.6 nho=20 nsp=0 exmin=-5 exmax=10 bas2=0 JTOT=2.0 PARITY=1 lmax=2 /

!&JPSET bastype=2 exmin=0.01  exmax=41.8 nk=100 nbins=10 lmax=2  JTOT=1.0 PARITY=1 inc=1 ehat=F/
!&JPSET bastype=2 exmin=0.01  exmax=41.8 nk=100 nbins=10 lmax=2  JTOT=1.0 PARITY=1 inc=2 ehat=F/
!&JPSET bastype=2 exmin=0.01  exmax=41.8 nk=100 nbins=20 lmax=0  JTOT=0.0 PARITY=1  /
!&JPSET bastype=2 exmin=0.01  exmax=41.8 nk=100 nbins=10 lmax=2  JTOT=2.0 PARITY=1 inc=1 ehat=F/
!&JPSET bastype=2 exmin=0.01  exmax=41.8 nk=100 nbins=10 lmax=2  JTOT=2.0 PARITY=1 inc=2 ehat=F/
!&JPSET bastype=2 exmin=0.01  exmax=41.8 nk=100 nbins=10 lmax=2  JTOT=3.0 PARITY=1  /
!  &JPSET bastype=1 mlst=4 gamma=1.5 bosc=1.6 nho=20 nsp=0 exmin=-5 exmax=10 bas2=0 JTOT=0.0 PARITY=1 lmax=0 /
!  &JPSET bastype=1 mlst=4 gamma=1.5 bosc=1.6 nho=20 nsp=0 exmin=-5 exmax=25 bas2=0 JTOT=0.0 PARITY=-1 lmax=1 /
!  &JPSET bastype=1 mlst=4 gamma=1.5 bosc=1.6 nho=20 nsp=0 exmin=0 exmax=25 bas2=0 JTOT=1.0 PARITY=-1 lmax=1 /  l=1 /
!  &JPSET bastype=1 mlst=4 gamma=1.5 bosc=1.6 nho=20 nsp=0 exmin=0 exmax=25 bas2=0 JTOT=2.0 PARITY=-1  lmax=3 /
!  &JPSET bastype=1 mlst=4 gamma=1.5 bosc=1.6 nho=20 nsp=0 exmin=0 exmax=25 bas2=0 JTOT=2.0 PARITY=1  lmax=2 /l=2 /
!&JPSET bastype=1 mlst=4 gamma=1.5 bosc=1.6 nho=20 nsp=0 exmin=0 exmax=25 bas2=0 JTOT=3.0 PARITY=1  lmax=2 /
 &JPSET /


 &SCATWF ifcont=f  emin=0.01 emax=10.01  nk=100 inc=1 jset=2  /

 &belambda  ifbel=f uwfgsfile="" lambda=1 nk=100 emin=0.1 emax=20 jset=2 /
 &bmlambda /

############################    REACTION       ####################################
&reaction elab=56   namep='d' mp=2.014 mt=12 namet='12C' zt=6 jt=0.0 / 
   

######################### COUPLING POTENTIALS FOR (X)CDCC ####################################
# (if no basis sets have been defined, this section will be skipped)
# ############################################################################################

&TRANS skip=T  rcc=4.614  writeff=F   /

&coupling qcmax=0  ncoul=0  lamax=8 /

&grid nquad=100  radmax=100.d0 rstep=0.1 rmax=100 rextrap=0 rvecin=0.1d0 drvec=0.025d0 hin=0.1 /

# Core-target (p+58Ni): KD
&corepotential ptype=1 ap=0 at=12  rc0=1.3
                  V0=-53.29 r0=1.124  a0=0.57  V0i=-0 r0i=1.17 a0i=0.675 / ! WS volume
&corepotential ptype=2 ap=0 at=12
                  V0i=-8.05 r0i=1.124 a0i=0.5 / ! WS deriv
&corepotential ptype=0 /


## Valence-target (n-58Ni):KD
&valencepotential ptype=1  ap=0 at=12  rc0=1.3
              V0=-52.25 r0=1.124  a0=0.57  V0i=-0 r0i=1.17 a0i=0.75 / ! WS volume.
&valencepotential ptype=2  ap=0 at=12  
             V0i=-8.05 r0i=1.124 a0i=0.5 / ! WS deriv
&valencepotential ptype=0 /



######################### SOLVE CC EQUATIONS ##############################
# method=0 PC-numerov, 
         1=ENA with 5 terms in Cosh[Sqrt[T]]
         2=ENA with 5 terms in Cosh[Sqrt[T]], only diagonal 
         3=Raynal
         4=Modified Numerov used in Fresco

&numerov hcm=0.05 rmaxcc=100  hort=5 method=4 jtmin=0 jtmax=50 skip=T /   


######################### TRIPLE DIFFERENTIAL CROSS SECTINS ##############################
&xsections fileamp="fort.137" thmin=0 thmax=180 dth=1  
           doublexs=F triplexs=F alphaxs=F   icore=1 ner=200 
           ermin=0.01 ermax=60 jsets(1:6)=F T T T T T/ T T T /

&framework sys='lab' idet=1 / 
&gridener Enlow=0.0 Enup=44.8 dEn=0.8 /      
&gridthetac tcl=0.0 tcu=180  dtc=1.0 /      
&gridthetav tvl=0.0 tvu=180  dtv=1.0  /
&gridphi phil=0.0 phiu=360 dphi=2.0 /
&polarization polar(1:3)=1.5d0 0d0 1.5d0 /

# ---------------------------------------  END OF INPUT ---------------------------------------------------



&reaction elab=80 jtmin=0 jtmax=40.0   mp=2.014 mt=58 zt=28 /     jbord(1:2)=0 50  jump(1:2)=2 5 / 








 

 &POTENTIAL part='Valence' a1=58  rc=1.
      V=44.70 vr0=1.192 a=0.663 w=3.95 wr0=1.192 aw=0.663 
      wd=5.32  wdr0=1.282 awd=0.550 /
 &POTENTIAL part='Core' a1=58 rc=1.25 
      V=39.50 vr0=1.192 a=0.663 w=3.97 wr0=1.192 aw=0.663
      wd=4.90  wdr0=1.278 awd=0.536 /
