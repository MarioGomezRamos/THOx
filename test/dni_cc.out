**************************************************
 ***        THOx+DCE+CC code: version 2.6        ***
**************************************************

 ***************** CONSTANTS **********************
 * hbarc=197.32706 MeV.fm     e^2= 1.43997 MeV.fm *
 * so, alpha= 1/137.03599       amu=931.4943 MeV  *
 **************************************************
   A real*8 is            8  bytes
   A complex*16 is           16  bytes
 Pre-reading input to set dimensions
 
  Dimensions for memory allocation
    o J/pi sets:           1  merged into           1
    o Maximum internal l:           2
 
  Pre-read finished
 - COMPOSITE:
     Core:    Ac=  1.0078  Zc=  1.00
     Valence: Av=  1.0087  Zv=  0.00

 - Reduced mass:  469.589 MeV/c2
 - CORE states:
 Changing kband=0 to kband=0.5
     # 1: J/pi= 0.5+   Ex= 0.00 MeV

  RADIAL GRID FOR INTERNAL WFS:
 - Using uniform grid with 2001 points:  [ Rmin= 0.000   Rmax= 100.0  Step= 0.050 fm ]

   ------------- Potential component: 1-------------

 o Undeformed valence-core potential: 
   COULOMB     : uniform charged sphere with Rc= 0.001 fm
   NUCLEAR part: Reid93 potential, ONLY for deuteron!!!!
 --------------------------------------------------
   
  PAULI forbidden sp states:
 => No states will be removed
  


 ******************************************
      BASIS SET:  1 WITH TOTAL J/pi= 1.0+
 ******************************************
          
 SINGLE-PARTICLE configurations:
   [ l val-core truncated at lmin,lmax= 0  2]
       S.p. configuration # 1 ( 0 0.5) 0.5
       S.p. configuration # 2 ( 2 0.5) 1.5
    
   
    CORE+VALENCE channels:
     Channel # 1 : Ic= 0.5 ( 0 0.5) 0.5 Ex= 0.000 MeV with sp config #1 and core #1
     Channel # 2 : Ic= 0.5 ( 2 0.5) 1.5 Ex= 0.000 MeV with sp config #2 and core #1
   
  ** Exact bound wf ** 
 Energy:   2.2200000000000002     
 Energy:   2.2287979446404647     
 Channel           1 (l,s,j,I,J)           0  0.50000000000000000       0.50000000000000000       0.50000000000000000        1.0000000000000000      Norm:  0.94294178401292106     
 Channel           2 (l,s,j,I,J)           2  0.50000000000000000        1.5000000000000000       0.50000000000000000        1.0000000000000000      Norm:   5.7058215880389201E-002
 Total norm:  0.99999999989331023     
 sqrt(rms):   3.9362081598751728     
 Energy:   2.2287979446404647      from   2.2200000000000002     


     ******  PROJECTILE SCATTERING STATES ******


     ******  REACTION CALCULATION ******

     Projectile: Mass=   2.014 amu   Z=   1.0
     Target:     Mass=  12.000 amu   Z=   6.0

     Nb. of target states:  1
     o state #  1   J/pi=0.0+    Ex= 0.000
     [using transition subroutine for spin-zero target]

     *********** COUPLING POTENTIALS  *********** 
     [No coupling potentials requested ]


     ******  CC SOLUTIONS ******
     [No reaction calculation requested]


  The following files have been created:
  40: Valence-core potentials.                137: Scattering amplitudes.                  
 200: .                                       300: All diff. x-sections.                   
 
 Timings:
